src/library/driftory.ts

```diff
@@ -22,14 +24,14 @@ const osdPromise = new Promise((resolve, reject) => {
 
 type Frame = any;
 type Container = any;
-type OnFrameChange = (params: { frameIndex?: number; isLastFrame: boolean }) => void;
+type OnFrameChange = (params: { frameIndex: number; isLastFrame: boolean }) => void;
 type OnComicLoad = (params: {}) => void;
 
 interface DriftoryArguments {
-  container: any;
-  onFrameChange: OnFrameChange;
-  onComicLoad: OnComicLoad;
-  prefixUrl: string;
+  container: HTMLElement;
+  onFrameChange?: OnFrameChange;
+  onComicLoad?: OnComicLoad;
+  prefixUrl?: string;
```

Comment:
Made `container: HTMLElement;` type for better autocomplete.

Made the rest of the arguments optional since they aren't necessary for the base functionality to work.